content/docs/tasks/telemetry/gateways/index.md

```diff
@@ -0,0 +1,632 @@
+---
+title: Remotely Accessing Telemetry Add-ons
+description: This task shows you how to configure external access to the set of Istio telemetry add-ons.
+weight: 99
+keywords: [telemetry,gateway,jaeger,zipkin,tracing,kiali,prometheus,add-ons]
+---
+
+This task shows how to configure Istio to expose and access the telemetry add-ons outside of
+a cluster.
+
+## Configuring Remote Access
+
+Access to the telemetry add-ons can be configured in a number of different ways. This task will cover
+two basic access methods: secure (via https) and insecure (via http). The secure method is *strongly
+recommended* for any production or sensitive environment. Insecure access is simpler to set up, but
+will not protect any credentials or data transmitted outside of your cluster.
+
+### Option 1: Secure Access (HTTPS)
+
+A server certificate is required for secure access. Follow these steps to install and configure
+server certificates for a domain that you control.
+
+It is possible to use self-signed certificates instead, but that is not covered by this task.
+[Securing Gateways with HTTPS Using Secret Discovery Service](/docs/tasks/traffic-management/secure-ingress/sds/)
+provides general details on using self-signed certificates to access in-cluster services.
+
+{{< warning >}}
+This task is concerned *solely* with securing the transport layer for access. When exposing the telemetry
+services externally, the services should be installed and configured to require authentication.
+{{< /warning >}}
+
+1. [Install Istio](/docs/setup/) in your cluster with `cert-manager` enabled and `istio-ingressgateway` configured to use
+the [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/secret#config-secret-discovery-service).
+
+    Use the following helm install options:
+
+    * `--set gateways.enabled=true`
+    * `--set gateways.istio-ingressgateway.enabled=true`
+    * `--set gateways.istio-ingressgateway.sds.enabled=true`
+    * `--set certmanager.enabled=true`
+    * `--set certmanager.email=mailbox@donotuseexample.com`
+
+    Install the telemetry add-ons using the following helm install options:
+
+    * Grafana: `--set grafana.enabled=true`
+    * Kiali: `--set kiali.enabled=true`
+    * Prometheus: `--set prometheus.enabled=true`
+    * Tracing: `--set tracing.enabled=true`
+
+1. Configure the DNS records for your domain.
+
+    1. Get the External IP Address of the `istio-ingressgateway`.
+
+        {{< text bash >}}
+        $ kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        <IP ADDRESS OF CLUSTER INGRESS>
+        {{< /text >}}
+
+    1. Set an environment variable to hold your target domain.
+
+        {{< text bash >}}
+        $ TELEMETRY_DOMAIN=<your.desired.domain>
+        {{< /text >}}
+
+    1. Point your desired domain at that External IP Address via your domain provider.
```

Comment:
How should the reader accomplish this? I assume the actual command depends on the provider. Could you provide an example?

Reply:
This is a topic that goes well beyond the scope of this task and varies from provider to provider. I've added some links to what I think are decently broad example providers FAQs, etc.